src/styles/index.scss

```diff
@@ -1,3 +1,152 @@
-body {
+.body {
   margin: 0;
 }
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+
+  &:hover {
+    background: #fff;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+  }
+}
+
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
The '.card' block should not have a fixed width set with 'width: 199px;'. It's better to use maximum widths or percentages to make the layout more responsive.